src/seedu/addressbook/data/person/Person.java

```diff
@@ -16,6 +16,8 @@
     private Address address;
 
     private final UniqueTagList tags;
+    private int sequenceNumber;
+    private static int nextSequenceNumber = 0;
```

Comment:
do take note of the semantics here. when we talk about nextSequenceNumber, usually we take this number for the object created. unless this is named `currentSequenceNumber` and set as 0, it should set to 1 instead.